### PR TITLE
TTP to display kernel info from last system boot

### PR DESCRIPTION
**Pull request by abhiabhi2306**
This will display all the information including error logs, bootloader module debug messages etc during the last system boot.

### CODE REVIEW

File: discovery/19d4a157-b388-44ff-9597-e61bfc44a3e7.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+description: |
+  This will display all the information including error logs, bootloader module debug messages etc during the last system boot.
+tactic: discovery
+technique:
+  id: T1082
+  name: System Information Discovery
+platforms:
+  darwin:
+    sh:
+      command: dmseg
```
Comment (khyberspache) [inline discovery/19d4a157-b388-44ff-9597-e61bfc44a3e7.yml:17]: `dmesg` output on OS X is terrible, let's drop this.

File: discovery/19d4a157-b388-44ff-9597-e61bfc44a3e7.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  linux:
+    sh:
+      command: dmesg
```
Comment (khyberspache) [inline discovery/19d4a157-b388-44ff-9597-e61bfc44a3e7.yml:20]: `dmesg`

File: discovery/19d4a157-b388-44ff-9597-e61bfc44a3e7.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+id: 19d4a157-b388-44ff-9597-e61bfc44a3e7
+metadata:
+  version: 1
+  authors: 
+    - Abhishek S (abhiabhi2306)
+  tags: []
+name: Display last system boot information
```
Comment (khyberspache) [inline discovery/19d4a157-b388-44ff-9597-e61bfc44a3e7.yml:7]: Print kernel diagnostic information

File: discovery/19d4a157-b388-44ff-9597-e61bfc44a3e7.yml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+description: |
+  This will display all the information including error logs, bootloader module debug messages etc during the last system boot.
```
Comment (khyberspache) [inline discovery/19d4a157-b388-44ff-9597-e61bfc44a3e7.yml:9]: Print the kernel diagnostic message buffer which includes bootloader information, debug messages, and more, from the current system boot.

**Review by khyberspache — CHANGES_REQUESTED** [review]

**Comment by abhiabhi2306** [comment]
Done, thank you :)

**Pull request merged**